documentation/library_guide/dynamic_selection_api/policies.rst

```diff
@@ -0,0 +1,133 @@
+Policies
+########
+
+The dynamic selection API is an experimental feature in the |onedpl_long| 
+(|onedpl_short|) that selects an *execution resource* based on a chosen 
+*selection policy*. There are several policies provided as part 
+of the API. Policies encapsulate the logic and any associated state needed 
+to make a selection. 
+
+Policy Traits
+-------------
+
+Traits can be used to determine useful type information about policies. 
+
+.. code:: cpp
+
+  namespace oneapi::dpl::experimental {
+  
+    template<typename Policy>
+    struct policy_traits{
+      using selection_type = typename std::decay_t<Policy>::selection_type;  
+      using resource_type = typename std::decay_t<Policy>::resource_type;
+      using wait_type = typename std::decay_t<Policy>::wait_type; //wait_type
+    };
+  
+    template<typename Policy>
+    using selection_t = typename policy_traits<Policy>::selection_type;
+  
+    template<typename Policy>
+    using resource_t = typename policy_traits<Policy>::resource_type;
+  
+    template<typename Policy>
+    using wait_t = typename policy_traits<Policy>::wait_type;
+  
+  }
+
+``selection_t<P>`` is the type returned by calls to ``select`` when using policy P. Calling ``unwrap`` on an object of type ``selection_t<P>`` returns an object of 
+type ``resource_t<P>``. When using the default SYCL backend, ``resource_t<P>`` 
+is ``sycl::queue`` and ``sycl::wait_t<P>`` is ``sycl::event``.  The user functions
+passed to ``submit`` and ``submit_and_wait`` are expected to have a signature of:
+
+.. code:: cpp
+
+  wait_t<P> user_function(resource_t<P>, ...);
+
+Common Reference Semantics
+--------------------------
+
+If a policy maintains state, the state is maintained separately for each 
```

Comment:
My suggestion:
If a policy maintains a state, the state is maintained separately for each 
independent policy instance So, for example, two independently constructed 
instances of a ``round_robin_policy`` will operate independently of each other. 
However, policies provide *common reference semantics*, so copies of a
policy instances are linked and share state.

Reply:
will make change without "linked"